expirationd.lua

```diff
@@ -166,6 +177,10 @@ local function worker_loop(task)
             end
         end
 
+        -- There is sleep and it can be long.
```

Comment:
Yes, it can be long. Please add more context to this comment. Currently it's a bit useless.

Reply:
That's ok? 
```lua
-- If we do not check the fiber for cancellation,
-- then the fiber may fall asleep for a long time, depending on `full_scan_delay`.
-- And a fiber that wants to stop this task can also freeze, a kind of deadlock.
```